packages/web-api/src/controllers/scan-request-controller.spec.ts

```diff
@@ -158,6 +158,7 @@ describe(ScanRequestController, () => {
 
             // tslint:disable-next-line: no-null-keyword
             loggerMock.setup(lm => lm.trackEvent('BatchScanRequestSubmitted', null, expectedMeasurements)).verifiable();
+            loggerMock.setup(lm => lm.trackMetric('QueuedScanRequests', 1, { priority: '1' })).verifiable();
```

Comment:
we need to discuss about passing priority as property for this metric.

Reply:
i see some concerns with this approach. if we go with this approach, 
if we queue 3 p "0" urls & 2 p "-10" urls, then we shou7ld raise the following metrics:
QueuedScanRequests - 3 (P 0)
QueuedScanRequests - 2 (P -10)

Given that we support -1000 to 1000 as priority value, this is not feasible.